pkg/collector/corechecks/cluster/helm/stub.go

```diff
@@ -0,0 +1,6 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2021 Datadog, Inc.
```

Comment:
```suggestion
// Copyright 2016-2022 Datadog, Inc.
```

Reply:
I've changed this to `2016-present` like we're doing in other files.